### PR TITLE
Add intro for development section

### PR DESCRIPTION
**Pull request by sales**
**Done in this PR**
- Add intro

@startae/frontend-team

### CODE REVIEW

File: source/development/index.md [hunk 1]
```diff
@@ -5,3 +5,5 @@ children: true
 ---
 
 # Development
+
+In this section we will describe our approach for the development process. Here you will find techniques for version controlling, tools, writing pull requests, reviewing code, pair programming, our styleguide and our technology stack.
```
Comment (romulomachado) [inline source/development/index.md]: In this section**,** we will describe our approach **to** the development process.

Comment (romulomachado) [inline source/development/index.md]: It features our styleguide, our technology stack along with techniques we use to develop better software.

Comment (Gabiras12) [inline source/development/index.md]: +:+1:

Comment (sales) [inline source/development/index.md]: Thank you guys. You're awesome

File: source/development/index.md [hunk 2]
```diff
@@ -5,3 +5,5 @@ children: true
 ---
 
 # Development
+
+In this section we will describe our approach to the development process. It features our styleguide, our technology stack along with techniques we use to develop better software.
```
Comment (renatocarvalho) [inline source/development/index.md]: Here's my suggestion. I removed a few words with the intention of being more direct. Added a few suggestions.

> Here's our approach to the development process. It features our style guide, our technology stack along with techniques and tools we use to develop a great software.

**Comment by sales** [comment]
Updated!

**Pull request merged**